src/fullscreenwindow.cpp

```diff
@@ -1,4 +1,4 @@
-#include <QMenu> // avoids "`QMenu` doesn't name a type" error in Qt6
+class QMenu;
```

Comment:
Please merge/squash this commit into the earlier one that introduced the changes that are now being fixed.